gemfire-shared/src/main/java/com/gemstone/gemfire/internal/shared/Version.java

```diff
@@ -227,11 +229,17 @@
   public static final Version GFXD_20 = new Version("GFXD", "2.0",
       (byte)2, (byte)0, (byte)0, (byte)0, GFXD_20_ORDINAL, GFE_75);
 
+  private static final byte GFE_80_ORDINAL = GFXD_14_ORDINAL;
+  public static final Version GFE_80 = new Version("GFE", "8.0", (byte)8,
+      (byte)0, (byte)0, (byte)0, GFE_80_ORDINAL);
+
   private static final byte GFXD_155_ORDINAL = 32;
 
   /** GemFire version is at 7.1 for compatibility with external clusters. */
   public static final Version GFXD_155 = new Version("GFXD", "1.5.5",
-      (byte)1, (byte)5, (byte)5, (byte)0, GFXD_155_ORDINAL, GFE_71);
+      (byte)1, (byte)5, (byte)5, (byte)0, GFXD_155_ORDINAL, GFE_80);
```

Comment:
GFE_71 on gemfire has putAll with 4 parts, while PutAll in Store has 5 parts.  PutAll in GFE8. on GemFire has 8 parts. 
GemFire does not have 7.5 .
Store did not have 8.0.